### DISCUSSION TITLE
Copy incorrect words list

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1351] [Ideas] [answered]
It would be nice to copy the word you get incorrect after a test for further examination later.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Something similar is being developed I believe #44

**Reply by unrealapex** [reply]
That idea is very similar, but I don't mean the words you get incorrect will be available globally like having a "commonly miss typed words statistic" on your account statistics, but after a test you have the option to see just the words you missed and be able to copy it.

**Reply by frogfrogfrogfrogfrog** [reply]
Ah that makes sense but I don't see the use in that @UnrealApex 

#44 still seems more useful in my opinion

**Comment by Miodec** [comment]
So, copy what you actually typed instead of copying what you were supposed to type? Not sure I see the usefulness of this.

**Reply by unrealapex** [reply]
Copying not what you typed, but the words you got incorrect.
